### DISCUSSION TITLE
Scroll position changes when clicked on any card.

### DISCUSSION THREAD

**Discussion by notKvS** [community/community#13157] [Other Feature Feedback, Questions, & Ideas] [answered]
Scroll position changes back to top when clicked on any card/feed repo/link in For you section and then when we come back again to the For you section, instead of storing the scroll position to last time where user was, it scroll backs to top of the For You section.

(Kind of annoying to scroll down again and again once I click on any card to explore)

**Comment by jfullstackdev** [comment, marked as answer]
one Staff already answered here

https://github.com/orgs/community/discussions/13485
